### DISCUSSION TITLE
CreatableSelect createFilter breaks create functionality

### DISCUSSION THREAD

**Discussion by wiebejordan** [JedWatson/react-select#4779] [Q&A]
I have a CreatableSelect component that works great for creating new options, but the default filtering is not desirable. When I implement createFilter, the filtering is exactly what I need, but it no longer allows me to create new options. Here is my code: 

```
const filterConfig = {
    ignoreCase: true,
    ignoreAccents: true,
    trim: true,
    matchFrom: "start",
  };

<CreatableSelect
        onChange={onChange}
        options={options}
        autosize
        value={value}
        key={key}
        isSearchable
        filterOption={createFilter(filterConfig)}
      />
```
How can this be resolved? Thank you!

**Comment by ebonow** [comment]
@wiebejordan You mentioned the inability to create new options after one has already been created, but I don't see `isMulti` as a prop so I wouldn't expect that you could create and select multiple options.

Is this the issue or is there some other behavior that I am unclear about?

https://codesandbox.io/s/goofy-platform-56wrm?file=/src/App.js
